README.md

```diff
@@ -91,33 +161,86 @@ client
 });
 ```
 
-Note that the return value for any asynchronous methods is the callback itself, so make sure to keep a reference to it and clear it according to its host lifecycle events. 
+> Note: The return value for any asynchronous methods is the Callback itself, making sure keeping a reference to it and clearing it according to its host lifecycle events is adviced. 
+
+If _RxJava_<sup><a href="https://github.com/ReactiveX/RxJava" title="a library for composing asynchronous and event-based programs using observable sequences for the Java VM."/>i</a></sup> is wanted instead, the `.observe()` method can be used to get an `Observable` instance:
 
-If you want to use RxJava instead, call the `observe()` method to get an `Observable` instance:
 ```java
 client
     .observe(CDAAsset.class)
     .one("jake")
     .subscribe(System.out::println);
 ```
 
-### Default Ordering
+Paging
+------
+
+If more then _100_ Resources are in the Space, `.fetchAll()` will only return the first _100_. If more Resources are needed, specify the limit with the `.limit(X)` like so:
+
+```java
+CDAArray result = 
+  client
+    .fetch(CDAEntry.class)
+    .limit(1000)
+    .all();
+```
+
+The maximum number of Resources to be requested is _1000_. 
 
-Bear in mind that there is no default ordering included for any method which returns a `CDAArray` instance. This means that if you plan to page through more than 100 results with multiple requests, there is no guarantee that you will cover all entries. It is however possible to specify custom ordering:
+For more then _1000_ Resources `.skip(N)`, `.limit(L)` and `.orderBy(F)` methods are needed. By using `.skip(N)`, the first _N_ Resources are ignored and _L_, from `.limit(L)`, items are returned. 
+
+To guarantee ordering, the use of the `.orderBy` method is required: It enforces the _Array_<sup><a href="https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/collection-resources-and-pagination" title="A collection of Resources from Contentful. Containts meta information about number and limits of the Resources it contains."/>i</a></sup> to be in a predictable order. 
+
+The following code can be used to request all Entries:
```

Comment:
|Before|After|
|------|-------|
|The following code can be used| The following code is used|